### DISCUSSION TITLE
SMSNotifier will become a paid functionality ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#16472] [General] [answered]
Hi,

Ater updating to latest Yeti version 6.4, my previous custom integration of OVH as SMS provider stops working as I expected because the module have been reworked by devs.

So, I'm studying how to integrate with OVH again and I've noticed this mention regarding licence in /app/Integrations/SMSProvider/SMSAPI.php

````

 * The file is part of the paid functionality. Using the file is allowed only after purchasing a subscription.
 * File modification allowed only with the consent of the system producer.

````

Does it mean that SMSNotifier module is now a paid functionality and I can't use the SMSAPI integration anymore as reference to write my OVH script ?

Regards,

**Comment by rskrzypczak** [comment, marked as answer]
> Does it mean that SMSNotifier module is now a paid functionality

No. The SMSNotifer module is free of charge and will remain so. Integrations with some SMS service providers may be paid, but you can always write your own integrations, which I think you want to do.
You only need to create your own provider file, inheriting from \App\Integrations\SMSProvider\Provider and add the functions required for your integration.
